package.json

```diff
@@ -48,6 +48,7 @@
     "typescript": "^3.4.3"
   },
   "devDependencies": {
-    "require-npm4-to-publish": "^1.0.0"
+    "require-npm4-to-publish": "^1.0.0",
+    "tslint-no-unused-expression-chai": "^0.1.4"
```

Comment:
This would be a runtime dependency in the resin-lint context

Reply:
Ah, that is correct

Reply:
Should be outdated, not sure why it's not showing as outdated